model-optimizer/extensions/back/ReverseInputChannels.py

```diff
@@ -265,8 +293,41 @@ class ReverseChannelsPropagationUp(BackReplacementPattern):
         'Subtract': lambda node, rc: ReverseChannelsPropagationUp.lift_up_through_eltwise(node, rc),
         'Pow': lambda node, rc: ReverseChannelsPropagationUp.lift_up_through_eltwise(node, rc),
         'Convert': lambda node, rc: ReverseChannelsPropagationUp.lift_up_through_eltwise(node, rc),
+
+        'Pad': lambda node, rc: ReverseChannelsPropagationUp.lift_up_through(node, rc),
     }
 
+    @staticmethod
+    def lift_up_through(node: Node, reverse_channels: Node):
+        r"""
+        BEFORE                       AFTER
+
+                                     previous_op
+                                          \
+        previous_op  begin end       ReverseChannels   begin end
+                 \     /  /                        \    /   /
+                   Pad                               Pad
+                    |                                 |
+              ReverseChannels                      next_op
+                    |
+                 next_op
+
+        returns boolean value whatever we should continue propagating current ReverseChannels operation up or not
+        """
+        if node.is_in_port_connected(0):
+            port = node.in_port(0)
```

Comment:
Please use more meaningful variable names. For example, `port -> node_input_port_0`